### PR TITLE
Change port configuration for contact point discovery #114

### PR DESCRIPTION
**Pull request by jedossa**
Change management port according to deployment configuration.

### CODE REVIEW

File: bootstrap-joining-demo/kubernetes-api/src/main/resources/application.conf [hunk 1]
```diff
@@ -6,8 +6,13 @@ akka {
 
   discovery {
     # pick the discovery method you'd like to use:
-//    method = akka-dns
+    # method = akka-dns
     method = kubernetes-api
+
+    kubernetes-api {
+      pod-label-selector = "app=appka"
```
Comment (longshorej) [inline bootstrap-joining-demo/kubernetes-api/src/main/resources/application.conf]: For this example, I think we should add labels for `actorSystemName` and use that. So, `pod-label-selector=actorSystemName=%s`, and add a label to the pod `actorSystemName: appka`.

Also, would you mind removing line 9 (the commented out akka-dns)?

File: bootstrap-joining-demo/kubernetes-api/src/main/resources/application.conf [hunk 2]
```diff
@@ -6,8 +6,13 @@ akka {
...
+      pod-port-name = "http"
```
Comment (longshorej) [inline bootstrap-joining-demo/kubernetes-api/src/main/resources/application.conf]: Rather leave this out -- i.e. the defaults here for consistency with another project Lightbend is working on.

Comment (ktoso) [reply bootstrap-joining-demo/kubernetes-api/src/main/resources/application.conf]: you mean to remove the pod-port-name? The latter part of the sentence had even me confused...

Comment (longshorej) [reply bootstrap-joining-demo/kubernetes-api/src/main/resources/application.conf]: Sorry for being unclear - Yes, defaults should be sufficient, i.e. no need to declare `pod-port-name`.

**Comment by ktoso** [comment]
I THINK this is correct now... could you check @longshorej ?

File: bootstrap-joining-demo/kubernetes-api/kubernetes/akka-cluster.yml [hunk 3]
```diff
@@ -26,12 +26,11 @@ spec:
               fieldPath: status.podIP
         livenessProbe:
           tcpSocket:
-#            port: 2551
-            port: 19999
+            port: 8080
```
[the remark this answers is not in the document]
Comment (jedossa) [reply bootstrap-joining-demo/kubernetes-api/kubernetes/akka-cluster.yml]: I have not looked into the reason but if liveness port and management port does not are the same I get: `Unsupported Content-Type, supported: application/json` when probing for seed nodes.

**Comment by longshorej** [comment]
@jedossa sorry I haven't had a chance to look at this in the past few days. I'll make some time tomorrow. Thanks for the improvement!

**Comment by ktoso** [comment]
Thanks!

**Pull request merged**